checkov/sca_image/runner.py

```diff
@@ -20,11 +20,13 @@ class Runner(PackageRunner):
     check_type = CheckType.SCA_IMAGE
 
     def __init__(self) -> None:
+        super().__init__()
         self._check_class: Optional[str] = None
         self._code_repo_path: Optional[Path] = None
         self._check_class = f"{image_scanner.__module__}.{image_scanner.__class__.__qualname__}"
         self.raw_report: Optional[Dict[str, Any]] = None
         self.image_referencers: Optional[ImageReferencer] = None
+        self.file_names = ['Dockerfile']
```

Comment:
@mikeurbanski1 dockerfile has a mask / regex to identify all file names

Reply:
Ah, yeah, I did it correctly in one place but not in this one :|